### DISCUSSION TITLE
Long Waits No Acces To Copilot Yet......

### DISCUSSION THREAD

**Discussion by Zatef1** [community/community#6947] [Copilot Conversations] [answered]
I have been waiting for a while to get the copilot I am a student and I am trying to learn **js** and I am always checking my email daily and all that I get is nothing please make the response to have access to the API faster! I also can assure you that many people are applying but GitHub can use some ai learning to accept some, i have to learn js by the end of this week, Github Please help us! aka me

**Comment by davecheney** [comment, marked as answer]
I'm sorry about the wait, please see https://github.com/github/feedback/discussions/6878

**Comment by Of-qinhai** [comment]
I have applied for more than 2 weeks and am still waiting for a reply.I really want to experience it quickly.

**Reply by martin-marko** [reply]
I've applied within 10 minutes of the technical preview announcement and just got in yesterday

**Reply by Of-qinhai** [reply]
... maybe random selection

**Comment by AiFind** [comment]
So...
